modules/plugins/jboss-as-7/src/main/java/org/rhq/modules/plugins/jbossas7/JBossProductType.java

```diff
@@ -143,6 +144,8 @@ private static JBossProductType determineJBossProductTypeViaProductConfFile(File
                 productType = JBossProductType.BPMS;
             } else if (slot.equals("dv")) {
                 productType = JBossProductType.JDV;
+            } else if (slot.equals("wildfly-full")) {
```

Comment:
Is this correct? I think `-full` denotes the profile but not the server version. I think it may be required to also check the api version or such.
@burmanm wdyt?